source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,134 @@
+=======================
+Time Series Collections
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about **time series collections** in
+MongoDB, and how to interact with them in the MongoDB Go driver.
+
+Time series collections efficiently store sequences of measurements over
+a period of time. The collection consists of time series data with the
+following information:
+
+- Data collected over time
+- Metadata that describes the measurement
+- Date of the measurement
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Example
+     - Measurement
+     - Metadata
+
+   * - Sales Data
+     - Revenue
+     - Company
+
+   * - Infection Rates
+     - Amount of People Infected
+     - Location
+
+Create a Time Series Collection
+-------------------------------
+
+To create a time series collection, pass the following parameters to the
+``CreateCollection()`` function:
+
+- The name of the new collection to create
+- The ``TimeSeriesOptions`` object specifying at least the time field
+
+.. important::
+
+   Versions prior to MongoDB 5.0 cannot create a time series collection.
```

Comment:
issue: Two items here, I think.

1. This should lead the section so people know the requirements.
2. I think rewording this may make it clearer as it's not clear what Versions is referring to until after reading more.
```suggestion
  Time series collections require MongoDB 5.0 or later.
```